lib/macho/load_commands.rb

```diff
@@ -141,6 +143,8 @@ module LoadCommands
       :LC_LINKER_OPTIMIZATION_HINT => "LinkeditDataCommand",
       :LC_VERSION_MIN_TVOS => "VersionMinCommand",
       :LC_VERSION_MIN_WATCHOS => "VersionMinCommand",
+      :LC_NOTE => "LoadCommand",
```

Comment:
Based on http://llvm.org/doxygen/Support_2MachO_8h_source.html, this is the corresponding structure:

```c
     struct note_command {
       uint32_t cmd;        // LC_NOTE
       uint32_t cmdsize;    // sizeof(struct note_command)
       char data_owner[16]; // owner name for this LC_NOTE
       uint64_t offset;     // file offset of this data
       uint64_t size;       // length of data region
  };
```

It looks like `LC_NOTE` is only being used in `MH_CORE` files, though, so this doens't need to be implemented in this PR :slightly_smiling_face:

Reply:
Yes, I haven't encountered it yet; I just added it since I was adding `0x32`, which I have encountered. 😄